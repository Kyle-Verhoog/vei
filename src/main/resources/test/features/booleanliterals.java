public class Booleanliterals {
  public Booleanliterals() {}
  protected boolean x = false;
}