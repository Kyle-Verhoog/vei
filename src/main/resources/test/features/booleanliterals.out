PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PROTECTED ┃ protected
BOOLEAN ┃ boolean
IDENTIFIER ┃ x
= ┃ =
BOOLEAN_LITERAL ┃ false
; ┃ ;
} ┃ }